### ISSUE TITLE
class c and ttgo lora 32 support?

### ISSUE THREAD

**Issue by halukmy** [HelTecAutomation/ESP32_LoRaWAN#31]
is it available class c and ttgo support?

thanks

**Comment by Heltec-Aaron-Lee** [comment]
Yes, I just updated this library, you can fix the micro definition in the ino file.
![image](https://user-images.githubusercontent.com/32860565/74146111-57521500-4c3b-11ea-91dc-18d44bec0f4c.png)

Because of this library need a unique license relevant to ESP32's ChipID, so it can't work with TTGO's hardware.
